extensions/xcode/package.json

```diff
@@ -251,7 +251,7 @@
           "name": "url",
           "placeholder": "URL to open",
```

Comment:
We might want to change this placeholder to an example such as in `XcodeSimulatorOpenUrlForm`, WDYT?

Reply:
Updated with https://github.com/qeude/extensions/pull/1/commits/c2de4ab286c5a9a60daa394327099a8bdf696ec9